systemtests/src/test/java/io/enmasse/systemtest/isolated/CommonTest.java

```diff
@@ -183,28 +183,33 @@ void testRestartComponents() throws Exception {
         List<Pod> pods = kubernetes.listPods();
         int runningPodsBefore = pods.size();
         log.info("Number of running pods before restarting any: {}", runningPodsBefore);
+        try {
+            for (Label label : labels) {
+                log.info("Restarting {}", label.labelValue);
+                KubeCMDClient.deletePodByLabel(label.getLabelName(), label.getLabelValue());
+                Thread.sleep(30_000);
```

Comment:
Why is it waiting?

Reply:
don't really know, let's see how it behaves without it